packages/logs/src/domain/trackNetworkError.ts

```diff
@@ -67,3 +119,67 @@ function format(type: RequestType) {
   }
   return 'Fetch'
 }
+
+/**
+ * Read bytes from a ReadableStream until `limit` bytes have been read.
+ */
+function readBytes(
+  stream: ReadableStream<Uint8Array>,
+  limit: number,
+  callback: (error?: Error, bytes?: Uint8Array) => void
+) {
+  const reader = stream.getReader()
+  const partialBuffers: Uint8Array[] = []
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**  to reuse [mdn](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams) naming‏
```suggestion
  const chunks: Uint8Array[] = []
```